Eatery/Controllers/Eateries/EateriesViewController.swift

```diff
@@ -210,6 +211,25 @@ class EateriesViewController: UIViewController, MenuButtonsDelegate, CLLocationM
         view.addGestureRecognizer(collectionView.panGestureRecognizer)
     }
     
+    func presentGivingDayPopup(){
+        let todayDate = Date()
+        let givingDayStart : Date = {
+            let formatter = DateFormatter()
+            formatter.dateFormat = "MM/dd/yy HH:mm"
+            return formatter.date(from: "03/14/19 00:00")!
+        }()
+        let givingDayEnd : Date = {
+            let formatter = DateFormatter()
+            formatter.dateFormat = "MM/dd/yy HH:mm"
+            return formatter.date(from: "03/14/19 23:59")!
+        }()
+        
+        if(UserDefaults.standard.bool(forKey: "didDonatePopup") == false && todayDate > givingDayStart && todayDate < givingDayEnd){
```

Comment:
you can do `!UserDefaults.standard.bool....`. No need to do == false

Reply:
Remove parens